src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/Sprint.java

```diff
@@ -14,10 +15,39 @@
 import net.ccbluex.liquidbounce.utils.Rotation;
 import net.ccbluex.liquidbounce.utils.RotationUtils;
 import net.ccbluex.liquidbounce.value.BoolValue;
+import net.ccbluex.liquidbounce.value.ListValue;
+import net.minecraft.client.settings.KeyBinding;
 import net.minecraft.potion.Potion;
 
 @ModuleInfo(name = "Sprint", description = "Automatically sprints all the time.", category = ModuleCategory.MOVEMENT)
 public class Sprint extends Module {
+    public final ListValue modeValue = new ListValue("Mode", new String[] {"Legit", "Vanilla"}, "Vanilla") {
+        protected void setSupported(final String value) {
+            if (value.toLowerCase() == "legit") {
```

Comment:
Please do not compare strings with == in Java! Never ever do this. Use .equals and especially rather use .equalsIgnoreCase instead of .toLowerCase() to compare the String while ignoring if the chars are upper or lower case.